alpine/Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM openjdk:8u201-alpine
+
+LABEL maintainer="Sonatype <cloud-ops@sonatype.com>"
+
+LABEL vendor=Sonatype \
+  com.sonatype.license="Apache License, Version 2.0" \
+  com.sonatype.name="Nexus IQ Server image"
+
+# This is where we will store persistent data
+VOLUME /sonatype-work
+
+# Create needed directories
+RUN mkdir -p /opt/sonatype/nexus-iq-server /etc/nexus-iq-server /var/log/nexus-iq-server
+
+# Change working dir
+WORKDIR /opt/sonatype/nexus-iq-server
+
+# Add nexus user and group
+RUN adduser -s /bin/false -S nexus nexus
+RUN addgroup nexus
+RUN chown nexus:nexus /opt/sonatype/nexus-iq-server /etc/nexus-iq-server /var/log/nexus-iq-server
+
+# Change to nexus user
+USER nexus
+
+# Version to download
+ARG IQ_SERVER_VERSION=1.64.0-02
+ARG IQ_SERVER_SHA256=d79cae556c60982191a026d8407ca4c8b96f26876d088067fa4ae7ef1c39cb99
+
+# Fetch and configure IQ
+RUN mkdir /tmp/work && cd /tmp/work \
+&& wget https://download.sonatype.com/clm/server/nexus-iq-server-${IQ_SERVER_VERSION}-bundle.tar.gz \
```

Comment:
So, I started thinking that this might be a case for using the `ADD` command (it isn't because it only extracts local files), but then I got to wondering why this is here and not part of https://github.com/sonatype/insight-brain/tree/master/nexus-iq-server

Not sure what the back story is, but seems like a good case for using one of the maven docker integrations to just build the image as part of the maven build.